docs/content/python/getting-started/application/index.md

```diff
@@ -56,51 +47,48 @@ $ sqlite3 expenses.db ".read expenses.sql"
-
-Additionally, there are two main places where we want to enforce our authorization logic: at the request/API layer, and at the data access layer.
-
-The goal of the former is to restrict which _actions_ a user can take in your application, e.g. are they allowed to fetch the expenses report via the `GET /expenses/report` route.
-
-The goal of the latter is to restrict them from viewing data they
-shouldn't have access to, e.g. they should not be able to see other users' data.
+In Oso, your authorization rules are expressed in a policy file, written in our language Polar.
+When someone tries to access a resource in your app, your app will need to query Oso to check if that access is allowed.
+That means adding Oso to your app has two parts: writing the policy file and writing the access check.
 
-### Add Oso
+In our sample application, we're storing our policies in the `authorization.polar` file, and all the authorization in the application is managed through the `authorization.py` file.
+Your app can take the same structure.
+(When starting out, it's reasonable to capture all policy logic in a single `authorization.polar` file, as we've done here.
+As your app grows, you can break that into many files, depending on your app's needs.)
 
-In our sample application, we are storing our policies in the `authorization.polar`
-file, and all of the authorization in the application is managed through the
-`authorization.py` file.
-
-In the application, we need to:
+### Initializing Oso
+In an application, we need to:
 
 1. Create the Oso instance
 2. Load in policy files.
 3. [Register application classes](getting-started/policies#application-types)
-4. Attach the Oso instance to the application
+4. Attach the Oso instance to the application.
 
-We have achieved this using the `init_oso` method:
+We'll write an `init_oso` method to call at app boot time:
 
 {{< literalInclude path="examples/python/getting-started/application/expenses-flask/app/authorization.py"
                    from="start-init" >}}
 
-We can now access this `oso` instance anywhere in our application, and specify
-which policy files are loaded in the application configuration.
+We can now access this `oso` instance anywhere in our application and specify which policy files are loaded in the application configuration.
+
+There are two main places where we want to enforce our authorization logic: at the request/API layer and at the data access layer.
+
+The goal of enforcement at the request layer is to restrict which _actions_ a user can take in your application.
+For instance, is a user allowed to fetch the expenses report via the `GET /expenses/report` route?
+
+The goal of enforcement at the data layer is to restrict users from viewing data they shouldn't have access to
+For instance, they shouldn't be able to see other users' data.
+
+
 
 ### Authorizing Routes
```

Comment:
I thought we were dropping the authorizing routes section? And why have we instead added an authorization requests section?